spacy/lang/pl/lemmatizer/trie.py

```diff
@@ -0,0 +1,75 @@
+import string
+
+charset = string.ascii_lowercase + "ąężźćńółęąś"
```

Comment:
Dałbym to UPPERCASE tak jak inne globalne zmienne

Reply:
A to jest w ogóle niepotrzebne, my bad, było do generowania z regexów, good catch